SU-HSDO/suhumsci
# SHS-5877: Editors can create anchor links to Colorbands

# READY FOR REVIEW

## Summary
- Adds an `id` to the container of colorband paragraphs

## Need Review By (Date)
10/23

## Urgency
medium

## Steps to Test
1. Visit a page with "Colorband" components (or create a new one). You can use one of the following:
    - https://hs-colorful-upkqlcwztqtvm3dptlfkdwwrmvrflxy6.tugboatqa.com/components/color-bands
    - https://hs-traditional-upkqlcwztqtvm3dptlfkdwwrmvrflxy6.tugboatqa.com/components/color-bands
2. Inspect the page source code and search for `ptype-hs-clr-bnd`. This class is added to the colorband component wrapper `div`. You should also see in the `div` an `id` with the format `component-XYZ` (where `XYZ` is a unique numerical ID):
    <img width="802" alt="Screenshot 2024-10-15 at 11 43 25 AM" src="https://github.com/user-attachments/assets/826556fc-c208-4fb5-b6d5-e22407629a62">
3. Visit any other flexible page and search for other types of components. Confirm that the `id` is not present in any case. You can search for `ptype-` to find the component wrapper `div` elements. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208572389505173
